defaults/main.yml

```diff
@@ -29,6 +29,7 @@ patroni_restapi_username: ""
 patroni_restapi_password: ""
 
 patroni_dcs: etcd
+patroni_dcs_exist: false

```

Comment:
DCS (or distributed configuration store) is a singular noun, so would be correct to use `patroni_dcs_exists`